data/test_domains/test_domain_from_directory2/domain_1.yml

```diff
@@ -0,0 +1,16 @@
+version: "2.0"
```

Comment:
I always find it hard to name things but can we rename `test_domain_from_directory2` to something self explanatory like `multiple_domain_files` and also `domain_1.yml` to `games.yml`? I feel it makes it more clear, what do you think?

Reply:
Sure - there was already a `test_domain_from_directory` folder for another test, but I can rename the folder, and the filenames inside 👍